contracts/serverless-v2/Executors.sol

```diff
@@ -442,7 +442,16 @@ contract Executors is
     function _releaseExecutor(
         address _executor
     ) internal {
-        _postJobUpdate(_executor);
+        if(!executors[_executor].draining) {
+            // add node to the tree if its max job capacity was reached, else update the node
+            if(executors[_executor].stakeAmount >= MIN_STAKE_AMOUNT)
+                _upsert(_executor, uint64(executors[_executor].stakeAmount));
+            // remove node from tree if stake falls below min level
+            else
+                _deleteIfPresent(_executor);
+        }
+        
+        executors[_executor].activeJobs -= 1;
     }
 
     function _slashExecutor(
```

Comment:
review the slashing mechanism again. Current implementation is not aligned at all with what was discussed.

Reply:
pushed the updated code for slashing